src/lightning/data/streaming/constants.py

```diff
@@ -17,6 +17,7 @@
 _INDEX_FILENAME = "index.json"
 _DEFAULT_CHUNK_BYTES = 1 << 26  # 64M B
 _DEFAULT_FAST_DEV_RUN_ITEMS = 10
+_MINIMUM_DISK_SIZE = 50 * 1024 * 1024 * 1024  # 50GB
```

Comment:
It is either this
```suggestion
_MINIMUM_DISK_SIZE = 50 * 1024 * 1024 * 1024  # 50GiB
```
or this
```suggestion
_MINIMUM_DISK_SIZE = 50 * 1000 * 1000 * 1000  # 50GB
```
I think the latter is more human friendly. You can then also use 1e9 to express the powers

Reply:
done.